### ISSUE TITLE
Bayes classification

### ISSUE THREAD

**Issue by pasq-cat** [JuliaTrustworthyAI/ConformalPrediction.jl#126]
i have some doubts on how to correctly create a conformalized bayes classifier with laplaceredux.
Laplaceredux approximate the posterior predictive distribution with a multidimensional gaussian, so for each element in the calibration set we get the Approximated MAP vectot  f\mu and a  variance matrix. 

i am not sure how to use these results to  create a discrete conformal set.

**Comment by pat-alt** [comment]
Let's double-check in Murphy p. 360

**Comment by pasq-cat** [comment]
> Let's double-check in Murphy p. 360

yeah, i don't get why he integrate the product of the cdf with the pdf  and what it has to do with the computation of the predictive posterior distribution
![image](https://github.com/user-attachments/assets/02c95a5a-b805-48ca-a2cf-ea69a4ad850d)
